tests/dashboards/security/security_rbac_tests.py

```diff
@@ -129,6 +253,23 @@ def test_get_dashboards_list__user_get_only_published_permitted_dashboards(self)
         for dash in published_dashboards + not_published_dashboards:
             revoke_access_to_dashboard(dash, new_role)
 
+    def __arrange_only_published_permitted(self):
```

Comment:
Is double underscore a new convention for private methods? The method name is also a little unclear. How about `def _create_sample_dashboards_with_role_access`

Reply:
my bad on the double underscore.
the name you're suggesting will not indicate its meaning 
the purpose of this method is to arrange a scenario matching to tests using it

Reply:
Sure, naming is hard... I think in this case the intent is very clear, but it would be nice to add a docstring in the future if the function name is not very straightforward.